src/foam/nanos/auth/User.js

```diff
@@ -655,12 +671,38 @@ foam.RELATIONSHIP({
   }
 });
 
+foam.RELATIONSHIP({
```

Comment:
Move these Relationships to   foam/nanos/auth/Relationships.js

Reply:
Done